src/distraction/DistractionSettings.java

```diff
@@ -0,0 +1,169 @@
+
+package distraction;
+
+/**
+ *
+ * @author Johnny
+ */
+public class DistractionSettings {
+    
+    public static boolean Rain;
+    public static boolean Snow;
+    public static boolean Fog;
+    public static boolean Sound;
+    public static boolean Box;
+    public static boolean Text;
+    public static boolean Pedestrian;
+    public static boolean Dark;
+    public static float intensityRain;
+    public static float intensitySnow;
+    public static float intensityFog;
+    public static float propabilitySound;
+    public static float propabilityBox;
+    public static float propabilityText;
+    public static float propabilityPedestrian;
+    public static float propabilityDark;
+    public static boolean distRunning;
```

Comment:
instead of all the settings for all subclasses of Distraction in one file, why not have the relevant settings in the (instance) of the specific class (e.g. DarkeningDistraction, etc...) ?